src/views/addProperty.js

```diff
@@ -155,7 +157,7 @@ export class AddProperty extends Component {
                                         </div> */}
                                         <div className="container-footer">
                                             <button className={`${isValid && "active"} save_button button is-rounded`} type="submit" disabled={isSubmitting}>SAVE</button>
-                                            <button className="button is-dark is-rounded" onClick={()=>{console.log("cancel pressed")}}>CANCEL</button>
+                                            <button className="button is-dark is-rounded" onClick={()=>{this.props.history.push('/manage/properties')}}>CANCEL</button>
```

Comment:
Because we're using React-Router, we could just use the <Link> component instead here https://reacttraining.com/react-router/web/api/Link

In fact, we should be doing this throughout the app where we have navigation buttons, but we're probably not very consistent yet 😄

Reply:
Totally, that is my bad. I'll revise it so it uses <Link> instead of the way I have it written. I agree, <Link> is much more straight forward, I'd just seen someone use the 'this.props.history.push' method somewhere else in the code :)